runtime/server_header.go

```diff
@@ -83,3 +82,57 @@ func (zh ServerHTTPHeader) Keys() []string {
 	}
 	return keys
 }
+
+// ServerTChannelHeader wrapper to implement zanzibar Header interface
+// on map[string]string
+type ServerTChannelHeader map[string]string
+
+// Get retrieves the string value stored on a given header. Bool
+// return value is used to distinguish between the presence of a
+// header with golang's zerovalue string and the absence of the string.
+func (th ServerTChannelHeader) Get(
+	key string,
+) (string, bool) {
+	// TODO: Canonicalize strings before lookup.
```

Comment:
I'd rather not do that. The correct string literal should be used here with the correct casing.